folium/map.py

```diff
@@ -246,6 +249,26 @@ def _repr_html_(self, **kwargs):
             out = self._parent._repr_html_(**kwargs)
         return out
 
+    def _repr_png_(self):
+        """Displays the PNG Map in a Jupyter notebook."""
+        # https://github.com/ipython/ipython/issues/7899#issuecomment-76541102
+        if self._png_image is None:
+            import selenium.webdriver
+
+            with tempfile.NamedTemporaryFile(suffix=".html") as f:
+                fname = f.name
+                self.save(fname)
+                driver = selenium.webdriver.PhantomJS(service_log_path=os.path.devnull)
+                driver.get('file://{}'.format(fname))
+                driver.maximize_window()
+                # Ignore user map size.
+                driver.execute_script("document.body.style.width = '100%';")
+                png = driver.get_screenshot_as_png()
```

Comment:
We probably want to "sleep" here for a few seconds to allow for the tiles to render.
Maybe that should be a `kwargs`.